microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -494,10 +494,23 @@ Whether to use local overrides for response headers.
 <!-- todo: need text -->
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 117.
+*  This checkbox is present in Microsoft Edge Canary 119.
 *  This checkbox is present in Microsoft Edge Stable 116.
 
 
+<!-- ====================================================================== -->
+## Enable "Delete all overrides" temporarily
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 119_
+
+<!-- todo: need text -->
```

Comment:
new Experimental Features checkbox in Canary 119:
* Enable "Delete all overrides" temporarily

Need sentence about this.  How does the UI change when this checkbox is selected?
@captainbrosset @zoherghadyali